pkgs/development/python-modules/diff_cover/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, jinja2, jinja2_pluralize, pygments,
+  six, inflect, mock, nose, coverage, pycodestyle, flake8, pyflakes, git,
+  pylint, pydocstyle }:
+
+buildPythonPackage rec {
+  pname = "diff_cover";
+  version = "1.0.2";
+
+  LC_ALL = "en_US.UTF-8";
+
+  src = fetchFromGitHub {
```

Comment:
any reason for not using PyPI?

Reply:
Package from PyPi fails tests when build is doing in sandbox. I told about that to author of diff-cover. He fixed that bug.

Here this conversation about this bug.
https://github.com/Bachmann1234/diff-cover/issues/72

Reply:
Good work! If it is a patch on top of a release, then we fetch the patch with `fetchpatch` instead and add it to `patches`.